docs/src/install-guide.md

```diff
@@ -1,112 +1,43 @@
 # Installation
 
-To install the latest version of Kani, run:
-
-```bash
-cargo install --locked kani-verifier
-cargo-kani setup
-```
+Kani offers an easy installation option on two platforms:
 
-This will build and place in `~/.cargo/bin` (in a typical environment) the `kani` and `cargo-kani` binaries.
-The second step (`cargo-kani setup`) will download the Kani compiler and other necessary dependencies (and place them under `~/.kani/`).
+* `x86_64-unknown-linux-gnu` (Most Linux distributions)
+* `x86_64-apple-darwin` (Intel Mac OS)
 
-Currently, only two platforms are supported:
+Other platforms are either not yet supported or require instead that you [build from source](build-from-source.md).
 
-* `x86_64-unknown-linux-gnu`
-* `x86_64-apple-darwin`
+## Dependencies
 
 The following must already be installed:
 
-* **Python version 3.8 or greater** and the package installer pip.
-* Rust installed via `rustup`
-* `ctags` is required for Kani's `--visualize` option to work correctly.
+* **Python version 3.6 or greater** and the package installer pip.
+* Rust installed via `rustup`.
+* `ctags` is required for Kani's `--visualize` option to work correctly. "Universal ctags" is recommended.
```

Comment:
I guess you used quotes on "Universal ctags" to avoid confusion. Maybe we just use `universal-ctags` instead and link the command to their webpage? (i.e., [`universal-ctags`](https://ctags.io/))